### ISSUE TITLE
Issue with results when using ta.Depoly() and ta.Restore()

### ISSUE THREAD

**Issue by YUIWK** [autonomio/talos#241]
Sorry to ask for your help all the time... I really appreciate your help. Thanks!

Here is the thing. The results of hyperparameter tuning was pretty good. The val_loss and loss was satisfactory. After doing hyperparamter tuning I tried to use ta.Depoly() and ta.Restore() to predict output in a test data set. However, I found that runing the below codes would return totally uncorrect results which are totally different from actual values. As the below codes show, I also tried traning data set and validation data set but I still got uncorrect results which would cause val_loss and loss very large. I am wondering am I using wrong codes? Do you have an idea? 

Here are the codes. (The codes about hyperparameter tuning worked so don't show them.)
### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### 
ta.Deploy(h,'model_1',metric = 'val_loss');
model_1 = ta.Restore('model_1.zip')
Y_train_hat = model_1.model.predict(X_train) 
Y_test_hat = model_1.model.predict(X_test)
Y_val_hat = model_1.model.predict(X_val)
### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ###

**Comment by mikkokotila** [comment]
What do you mean by you get "wrong results"? What results do you expect to get?

**Comment by mikkokotila** [comment]
Closing here, feel free to open new issues is something arise.

**Comment by mikkokotila** [comment]
I've added [docstring](https://github.com/autonomio/talos/blob/daily-dev/talos/commands/deploy.py#L13) to Deploy(). Make sure to check it also.

**Comment by YUIWK** [comment]
Sorry for the late reply. I will check the results again with the use of asc=Ture. Thank you very much!

**Comment by YUIWK** [comment]
You are right. It's my fault! Thank you very much! Thank you!
